src/rules/compositeFieldsAreNullable.ts

```diff
@@ -23,8 +23,7 @@ function isAllowedNonNullCompositeField(
   exceptions: Array<string>,
 ): boolean {
   // exceptions
-  const isException = exceptions.includes(type.name);
-  if (isException === true) {
+  if (exceptions.includes(type.name) === true) {
```

Comment:
Does this need `=== true`? It can only return a boolean, right?

Reply:
Yeah, you are right, this flew over my radar.